content/blog/2019/secure-egress-control-istio/index.md

```diff
@@ -0,0 +1,323 @@
+it is tampering-proof.
+
+On top of these beneficial features, Istio egress traffic control provides additional advantages:
+
+*  It allows defining access policies in the same language for ingress/egress/in-cluster traffic. The cluster operators
+   need to learn a single policy and configuration language for all the kinds of traffic.
+*  Istio egress traffic control is integrated with Istio policy and telemetry adapters and can work out-of-the-box.
+*  When you use external monitoring/access control systems with Istio, you must write the adapters for them only once,
+   and then apply the adapters for all the kinds of traffic: ingress/egress/in-cluster.
+*  The Istio operators can apply Istio traffic management features to egress traffic, such as
+   load balancing, passive and active health checking, circuit breaker, timeouts, retries, fault injection, and others.
+
+We call a system that has the advantages above **Istio-aware**.
+
+Let me summarize the features of Istio egress traffic control and of the alternative solutions in the following table:
+
+| | Kubernetes Network Policies | Legacy Egress Proxy/Firewall | Istio Egress Traffic Control |
+| --- | --- | --- | ---|
+| DNS-aware | {{< cancel_icon >}} | {{< checkmark_icon >}} | {{< checkmark_icon >}} |
+| Kubernetes-aware | {{< checkmark_icon >}} | {{< cancel_icon >}} | {{< checkmark_icon >}} |
+| Transparent | {{< checkmark_icon >}} | {{< cancel_icon >}} | {{< checkmark_icon >}} |
+| Istio-aware | {{< cancel_icon >}} | {{< cancel_icon >}} | {{< checkmark_icon >}} |
+
+### Performance considerations
+
+Note that Istio egress control has its price, which is the increased latency of the calls to external services and
+increase of CPU usage by the cluster pods.
+After all, the traffic has to pass through two proxies, namely the sidecar proxy of the
+application and the proxy of the egress gateway. In the case of
+[TLS egress traffic to wildcard domains](/docs/examples/advanced-gateways/wildcard-egress-hosts/),
+you have to add
+[an additional proxy](/docs/examples/advanced-gateways/wildcard-egress-hosts/#wildcard-configuration-for-arbitrary-domains),
+making the count of proxies between the application and the external service three. The traffic between the second and
+third proxies is on the local network of the pod, so it should not have significant impact on the latency.
+
+See a [performance evaluation](/blog/2019/egress-performance/) of different configurations of Istio egress
+traffic control. I would encourage you to measure carefully different configurations for your own applications and your
+external services, and decide whether you can afford the performance overhead for your use cases. You should weigh the
+required level of security versus your performance requirements, and also compare with the performance overhead of
+alternative solutions.
+
+Let me provide our take on the performance overhead of Istio egress traffic control. First, we are working to reduce
```

Comment:
I'd reorganize this paragraph. First, explain why perf is pretty good as it is, and _then_ explain that "even though perf is pretty good, we believe we can further eliminate overhead with..."

Reply:
Addressed in https://github.com/istio/istio.io/pull/3179/commits/3cae0a0b2aaee6b3a36f746dcf50857bc8530fb1